### PR TITLE
Make a bulleted list complete

### PR DESCRIPTION
**Pull request by jpmedley**
Make a bulleted list complete so that someone scanning the page won't overlook the unbulleted item.

### CODE REVIEW

File: site/en/docs/extensions/mv3/content_scripts/index.md [hunk 1]
```diff
@@ -187,10 +187,10 @@ not well known or when content scripts should not always be injected on known ho
 Introduced in Chrome 96, dynamic declarations are similar to [static
 declarations][header-cs-static], but the content script object is registered with Chrome using
 methods in the [`chrome.scripting` namespace](/docs/extensions/reference/scripting/) rather than in
-[manifest.json][doc-manifest]. In addition to
-[registering][api-register-cs] content scripts, the Scripting API also allows extension developers
+[manifest.json][doc-manifest]. The Scripting API also allows extension developers
 to:
 
+- [registering][api-register-cs] content scripts.
```
Comment (AmySteam) [inline site/en/docs/extensions/mv3/content_scripts/index.md:193]:
```suggestion
- [Register][api-register-cs] content scripts.
```
Changed to match the rest of the list items

**Review by AmySteam — CHANGES_REQUESTED** [review]
Thanks Joe. I suggested a quick tweak to match the other list items better.

**Comment by jpmedley** [comment]
I should have caught that. Thanks.

**Review by AmySteam — APPROVED** [review]
👍

**Pull request merged**